webpack/components/extensions/HostDetails/ActionsBar/index.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import {
+  useSelector,
+} from 'react-redux';
+
+import { DropdownGroup, DropdownItem } from '@patternfly/react-core';
+import {
+  CubeIcon,
+} from '@patternfly/react-icons';
```

Comment:
```suggestion
import { useSelector } from 'react-redux';
import { DropdownGroup, DropdownItem } from '@patternfly/react-core';
import { CubeIcon } from '@patternfly/react-icons';
```

nit, can you use the prettier formatter in this file? :)